application-management/export-all-enterprise-apps-secrets-and-certs.ps1

```diff
@@ -10,19 +11,36 @@
 #In no event shall Microsoft, its authors, or anyone else involved in the creation, production, or delivery of the script be liable for any damages whatsoever (including, without limitation, damages for loss of business profits, business interruption, loss of business information, or other pecuniary loss) arising out of  the use of or inability to use the sample or documentation, even if Microsoft has been advised of the possibility of such damages.
 #################################################################################
 
-Connect-AzureAD
 
-#$Applications = Get-AzureADApplication -all $true
-$EnterpriseApps = Get-AzureADServicePrincipal -all $true
+<#
+get-MgServicePrincipal -ServicePrincipalId "0df2de66-3046-45a0-accf-7dd84477f179" | fl
+$appss = get-MgServicePrincipal -ServicePrincipalId "0df2de66-3046-45a0-accf-7dd84477f179"
+$appss.PasswordCredentials
+$appss.DisplayName
+$appss.Id
+$appss.KeyCredentials
+
+Get-MgServicePrincipalOwner -ServicePrincipalId "4098eac4-ae85-4505-997c-6cef1429c1cd" | fl
+$owwnerr = Get-MgServicePrincipalOwner -ServicePrincipalId "4098eac4-ae85-4505-997c-6cef1429c1cd"
+$owwnerr.AdditionalProperties.displayName
+$owwnerr.AdditionalProperties.userPrincipalName
+$owwnerr
+#>
```

Comment:
This should be removed - it looks like it contains (potentially) real account IDs.